rundeck/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "display_name": "Rundeck",
+  "maintainer": "forrest@rundeck.com",
+  "manifest_version": "1.0.0",
+  "name": "Rundeck",
+  "metric_prefix": "rundeck.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Automate Remediation Actions using Rundeck Webhooks",
+  "guid": "2e3492d3-14fd-409d-b339-fb1bc14d7db9",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Rundeck Integration",
+  "categories": [
+    "orchestration",
+    "notification"
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "rundeck",
+  "assets": {
+    "dashboards": {},
+    "saved_views": {},
+    "monitors": {},
+    "service_checks": {}
```

Comment:
There needs to be a service checks file even if it contains an empty list. CI is failing because of this.